### DISCUSSION TITLE
How to use Super Table with Side Panel

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#10655] [Help] [answered]
Hello, 
I use Super Table component then with event ROW CLICK, the side panel will show. Usually in table component of budibase to use side panel (Details Side Panel) use `{{Clicked.row.id}}`, but in Super Table there is nothing only {{ Row ID }}. How to use it?
Thanks.

Below i attached a screenshoot,

![Clicked Row](https://github.com/Budibase/budibase/assets/5665115/d0f7b31c-643f-4c7f-b952-b68b8f48c02c)

**Comment by muksidin** [comment]
Solved, i only define `_id` in Super Table Settings (Id Column)

**Reply by ConorWebb96** [reply, marked as answer]
Hey @muksidin,

I'd recommend reaching out to: https://github.com/poirazis as he is the one who made this plugin. He likely will be able to provide additional insight on this. I'm glad that you where able to figure this out.

Hopefully this helps.
